app/emails/dreamail.py

```diff
@@ -74,7 +74,7 @@ def send_dreamail(simulate, skip_audit_check):
             supplier_name=supplier.name
         )
 
-        subject = 'Please review your pricing and/or case studies on the Marketplace'
+        subject = 'Please review your pricing on the Marketplace'
```

Comment:
I don't think we should update the dreamail files as that was a once off.  These emails aren't being sent to sellers on an ongoing basis.

Reply:
Fixed the dreamail files back to the original content.